src/components/Table.tsx

```diff
@@ -181,10 +182,19 @@ export const Table = <TableRowData extends BaseTableRowData | CustomRowConfig>({
       if (column == null || column.allowsSorting === false) {
         return 0;
       }
+
       const first = (isCustomRow(a) ? 0 : column.getCellValue(a)) ?? undefined;
       const second = (isCustomRow(b) ? 0 : column.getCellValue(b)) ?? undefined;
       const sortDirectionAsNumber = sortDirection === 'descending' ? -1 : 1;
 
+      // Pinned check: show items that are isPinned first
+      const isFirstPinned = isTableRowData(a) && getIsRowPinned?.(a);
```

Comment:
can this just be called isFavorited instead?

also nit, i think it's more readable to pull out favorited markets into its own list, and then non-favorited markets later, and then merge the two arrays in order rather than shoving the global custom ordering into one sort function

Reply:
I think one sort function is much easier given the way this table is implemented.

Reply:
yeah makes sense

Reply:
Since this is the Table component it shouldn't have any concept of `isFavorited`. The generic `pinned` makes more sense to me here.